pulsar/internal/pulsartracing/producer_interceptor.go

```diff
@@ -33,7 +33,9 @@ func (t *ProducerInterceptor) BeforeSend(producer pulsar.Producer, message *puls
 	buildAndInjectSpan(message, producer).Finish()
 }
 
-func (t *ProducerInterceptor) OnSendAcknowledgement(producer pulsar.Producer, message *pulsar.ProducerMessage, msgID pulsar.MessageID) {
+func (t *ProducerInterceptor) OnSendAcknowledgement(producer pulsar.Producer,
```

Comment:
Can we revert these formatting changes since they have nothing to do with this change?

Reply:
This will cause Action CI to fail:

```
Error: pulsar/internal/pulsartracing/producer_interceptor.go:36: line is 136 characters (lll)
func (t *ProducerInterceptor) OnSendAcknowledgement(producer pulsar.Producer, message *pulsar.ProducerMessage, msgID pulsar.MessageID) {
Error: pulsar/internal/pulsartracing/producer_interceptor_test.go:58: line is 132 characters (lll)
func (p *mockProducer) SendAsync(context.Context, *pulsar.ProducerMessage, func(pulsar.MessageID, *pulsar.ProducerMessage, error)) {
Error: Process completed with exit code 1.
```